hopsworks-api/src/main/java/io/hops/hopsworks/api/zeppelin/socket/NotebookServer.java

```diff
@@ -157,52 +157,59 @@ public void onMessage(String msg, Session conn) {
     try {
       Message messagereceived = deserializeMessage(msg);
       LOG.log(Level.FINE, "RECEIVE << {0}", messagereceived.op);
-      LOG.log(Level.FINE, "RECEIVE PRINCIPAL << {0}", messagereceived.principal);
+      LOG.log(Level.FINE, "RECEIVE PRINCIPAL << {0}",
```

Comment:
@kouzant can you undo this refactoring? Set the right margin to 120 on your IDE.